backend/src/routes/db_lock/index.js

```diff
@@ -4,19 +4,19 @@ const what = "db_lock";
 const routes = [
   {
     method: "POST",
-    url: `/${what}`,
+    url: `/${what}/add`,
     // schema: validators.addOne,
     handler: controller.addLockByParams,
   },
   {
-    method: "GET",
-    url: `/${what}`,
+    method: "POST",
+    url: `/${what}/get`,
     // schema: validators.getOne,
     handler: controller.getLockByParams,
   },
   {
-    method: "DELETE",
-    url: `/${what}/:id`,
+    method: "POST",
+    url: `/${what}/delete`,
```

Comment:
If the GET request returned the id of the db lock row we could keep this as a DELETE `/db_lock/:id`.